tests/eclient/eden+eclient.sh

```diff
@@ -7,22 +7,11 @@ EDEN=./eden
 if $EDEN config get --key eve.hostfwd | grep -v 2223
```

Comment:
This file should not be here at all as in any other test

Reply:
https://github.com/lf-edge/eden/pull/337#issuecomment-716670577